app/services/arclight_repository_mapper.rb

```diff
@@ -4,27 +4,29 @@
 class ArclightRepositoryMapper
   class ArclightRepositoryMapperError < StandardError; end
 
-  attr_reader :aspace_repository_code, :ead_id
+  attr_reader :aspace_repository_code, :file_identifier
 
-  def self.map_to_code(aspace_repository_code:, ead_id:)
-    new(aspace_repository_code:, ead_id:).map_to_code
+  def self.map_to_code(aspace_repository_code:, file_identifier:)
+    new(aspace_repository_code:, file_identifier:).map_to_code
   end
 
-  def initialize(aspace_repository_code:, ead_id:)
+  def initialize(aspace_repository_code:, file_identifier:)
     @aspace_repository_code = aspace_repository_code
-    @ead_id = ead_id
+    @file_identifier = file_identifier
   end
 
   def map_to_code
     code = if aspace_repository_code == 'speccoll'
-             code_from_ead_id
+             # Fall back on manuscripts
+             # if we can't determine the sub-category of speccoll from the syntax of the file_identifier
+             file_identifier ? code_from_file_identifier : 'manuscripts'
```

Comment:
We don't need this anymore, right? Since we'll raise `AspaceResourceError` before we get here if we don't have a file identifier?